src/derivative_operators/concretization.jl

```diff
@@ -163,190 +163,190 @@ function Base.convert(::Type{AbstractMatrix},A::AbstractBC{T}) where T
 end
 
 # Multidimensional BC operators
-_concretize(Q::MultiDimDirectionalBC, M) = _concretize(Q.BCs, M)
+# _concretize(Q::MultiDimDirectionalBC, M) = _concretize(Q.BCs, M)

```

Comment:
what's the reason for commenting instead of removing?

Reply:
Earlier it was an Affine operator with concretizations. Now it isn't and hence won't support this but rather than removing I thought about commenting out keeping in mind that if by any means one can come up with a way to still concretize this later. 
Now when I think of it, I don't know why this should necessarily be manipulated and concretized  ┐(•_•)┌